src/ofono/provider_ofono.cpp

```diff
@@ -221,7 +223,8 @@ static constexpr const char *property_names[] = {
     "MMSContext",
     "DataRoamingAllowed",
     "GPRSAttached",
-    "VoiceCall"
```

Comment:
No, afaik nobody uses it yet, so I just renamed until someboby are going to do it :)